assets/src/edit-story/components/canvas/singleSelectionMovable.js

```diff
@@ -263,9 +263,11 @@ function SingleSelectionMovable({ selectedElement, targetEl, pushEvent }) {
       snapHorizontal={true}
       snapVertical={true}
       snapCenter={true}
-      horizontalGuidelines={[0, canvasHeight / 2, canvasHeight]}
-      verticalGuidelines={[0, canvasWidth / 2, canvasWidth]}
-      elementGuidelines={otherNodes}
+      horizontalGuidelines={
+        actionsEnabled && [0, canvasHeight / 2, canvasHeight]
+      }
+      verticalGuidelines={actionsEnabled && [0, canvasWidth / 2, canvasWidth]}
```

Comment:
I think these changes are all correct one, but there's one more issue. The guidelines show up because of the `dragStart` call [above](https://github.com/google/web-stories-wp/blob/4afcdcba7610906c72d9537fbe4472204375be9b/assets/src/edit-story/components/canvas/singleSelectionMovable.js#L87). Let's also condition that call by `!isBackrgoundELement`.

Reply:
I used the same `actionsEnabled` condition to also cover `isFill`. Didn't see a reason why not.

Reply:
👍